config/environment.rb

```diff
@@ -2,14 +2,4 @@
 require_relative 'application'
 
 # Initialize the Rails application.
-Rails.application.initialize!
-ActionMailer::Base.delivery_method = :smtp
-ActionMailer::Base.smtp_settings = {
-  address:              'smtp.gmail.com',
-  port:                 587,
-  domain:               'gmail.com',
-  user_name:            ENV['MAILER_EMAIL_ADDRESS'],
-  password:             ENV['MAILER_EMAIL_PASSWORD'],
-  authentication:       :plain,
-  enable_starttls_auto: true
-}
\ No newline at end of file
+Rails.application.initialize!
```

Comment:
Where did this config move to? Just for my understanding

Reply:
we already have it inside environments/production, and for test/development we use letter_opener.